rosetta-source/src/main/rosetta/product-template-type.rosetta

```diff
@@ -23,6 +23,10 @@ import cdm.product.collateral.*
 
 import cdm.mapping.config.*
 
+type TransferableProduct:  <"A TransferableProduct is a type of Product which can be used in a SettlementCommitment for a basic cash settled trade of either an Asset with or without the addition of specific EconomicTerms.">
```

Comment:
TransferableProduct can be used elsewhere as well if my understanding is correct, i.e. not just in a SettlementCommitment?

Reply:
Not in this version :-)
In Phase 3, this will be updated further